### PR TITLE
implementation of help icon

### LINKED ISSUE #134: The dropdown menu doesn't get closed
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

The dropdown menu doesn't closes while clicking on the white part of the webapp.

screen shot 2018-05-25 at 12 50 50

### LINKED ISSUE #187: Implementation of Help Icon
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

**Actual Behaviour**

We already have help icon for the email part.

**Expected Behaviour**

Implementing help icon for the rest of the remaining fields.

### PR DESCRIPTION
**Pull request by rpotter12**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- added question mark image in front of other option
- created function for pop-up box.

Fixes #187 
@meets2tarun I have also applied a method for solving dropdown issue(#134 and #181) but still, it is not solving. Also review these changes.

### CODE REVIEW

**Comment by adhavalboy** [comment]
@rpotter12 still not solved it show drop down anywhere on screen and question mark( logo) after email  is not poping an alert.

**Comment by meets2tarun** [comment]
@rpotter12 
Try to resolve one issue with one PR.
You can make more than 1 PR for multiple issues.

Please give a Heroku link to the changes you made in the PR.

Thanks

**Comment by rpotter12** [comment]
heroku link : https://rpotter.herokuapp.com/

File: templates/index.html [hunk 1]
```diff
@@ -20,6 +20,14 @@
   <title>Meilix ISO Generator</title>
 </head>
 <script type="text/javascript">
+	$(document).on('click', function() {
```
[the remark this answers is not in the document]
Comment (rpotter12) [reply templates/index.html]: @adhavalboy You are right but the issue is not solved yet. We have to check the changes locally also. Once the issue will be solved we will make a separate file for it.

[the remark this answers is not in the document]
Comment (adhavalboy) [reply templates/index.html]: okay :+1:

**Comment by meets2tarun** [comment]
@rpotter12 
Please update this branch with upstream master and then we would ship this.

Thanks

**Comment by rpotter12** [comment]
@meets2tarun I have updated my branch with upstream master.

**Pull request merged**